### ISSUE TITLE
Add support for pygame-ce

### ISSUE THREAD

**Issue by damusss** [chaquo/chaquopy#1278]
It'd be great if I could run pygame-ce (2.5.2) applications on a phone. Not pygame, I mean pygame-ce. Thank you if you consider it. If you tell me how, I could even try and make the PR myself, but I have no idea how to "support" a library.

**Comment by rmartin16** [comment]
FYI: I didn't follow all the details but someone at least somewhat affiliated with pygame-ce was trying to build it for Android; I'm not sure where it stands now though.

- https://github.com/beeware/briefcase/issues/1906

**Comment by mhsmith** [comment]
The instructions for building Android packages are [here](https://github.com/chaquo/chaquopy/blob/master/server/pypi/README.md). But a complete user interface framework like Pygame, Kivy or PyQT is likely to require significant changes to add Android support, which could only be done by someone familiar with the framework's internals.

**Comment by oddbookworm** [comment]
Yeah, I was never successful lol. I'm probably going to try again at some point, but I've got a lot going on that'll probably delay it
